src/tools/debugger/Makefile

```diff
@@ -20,7 +20,8 @@ OBJS =  mhpdbg.o mhpdbgc.o dis8086.o
 all: lib $(BINPATH)/bin/dosdebug
 
 dosdebug: dosdebug.c
-	$(CC) $(ALL_CFLAGS) $(ALL_LDFLAGS) $(ALL_CPPFLAGS) $< -o $@
+	$(CC) $(ALL_CFLAGS) $(ALL_LDFLAGS) $(ALL_CPPFLAGS) $< $(READLINE) -o $@
```

Comment:
Strange place for `$(READLINE)` - it should
be somewhere near `$(ALL_LDFLAGS)` (if not
inside them).

Reply:
Yes, may be true for libraries at least.